brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -355,30 +359,65 @@ private void configureConfigKeysSpec(EntitySpec spec, ConfigBag bag){
         }
         return propertyMap;
     }
+    
+    protected Map<String, Operation> getStandardInterfaceOperations() {
+        Map<String, Operation> operations = MutableMap.of();
 
+        if (indexedNodeTemplate.getInterfaces() != null) {
+            MutableMap<String, Interface> indexedNodeTemplateInterfaces =
+                    MutableMap.copyOf(indexedNodeTemplate.getInterfaces());
 
-    protected Map<String, Operation> getInterfaceOperations() {
-        final Map<String, Operation> operations = MutableMap.of();
+            Interface indexedNodeTemplateInterface = findInterfaceOfNodeTemplate(
+                    indexedNodeTemplateInterfaces, VALID_INTERFACE_NAMES);
 
-        if (indexedNodeTemplate.getInterfaces() != null) {
-            final ImmutableList<String> validInterfaceNames = ImmutableList.of("tosca.interfaces.node.lifecycle.Standard", "Standard", "standard");
-            final MutableMap<String, Interface> interfaces = MutableMap.copyOf(indexedNodeTemplate.getInterfaces());
-
-            for (String validInterfaceName : validInterfaceNames) {
-                Interface validInterface = interfaces.remove(validInterfaceName);
-                if (validInterface != null) {
-                    operations.putAll(validInterface.getOperations());
-                    if (!interfaces.isEmpty()) {
-                        log.warn("Could not translate some interfaces for " + this.nodeId + ": " + interfaces.keySet());
+
+            if (indexedNodeTemplateInterface != null) {
```

Comment:
`nodeTemplateInterface` is ignored if `indexedNodeTemplateInterface` is null. Is it possible for `indexedNodeTemplateInterface` to be null but for `nodeTemplateInterface` to be non-null?